demos/README.md

```diff
@@ -0,0 +1,92 @@
+## Processing scripts for Vitessce demo datasets
+
+Previously, we developed custom processing scripts for Vitessce demo data in the `vitessce-data` repository. 
+
+However, now that there are consensus single-cell file formats (e.g., h5ad) and an ecosystem of data processing packages in the community, we aim to leverage those. Writing demo dataset processing code using Scanpy and AnnData in small Snakemake workflows in this repository should allow us to iterate more quickly and share more demos.
+
+### Setup
+
+Set up the `vitessce-python-demos` environment using conda.
+
+```sh
+cd demos
+conda env create -f environment.yml
+conda activate vitessce-python-demos
+pip install -e "..[testing]"
+```
+
+### Run
+
+```sh
+snakemake --cores all --rerun-triggers mtime
+```
+
+### Serve data locally
+
+```sh
+http-server --cors='*' --port 8000 .
+```
+
+### Add a new demo
+
+To add a new demo, run
+
+```sh
+./create_demo.sh {new_demo_dirname}
+```
+
+Then,
+- Add a subworkflow entry to the main demos workflow in `./Snakefile`
+- Specify the output filenames in the `output` list in `./{new_demo_dir}/config.yml`
+- Fill in the Snakefile in `./{new_demo_dir}/Snakefile`
+- Write a test Vitessce configuration in `./{new_demo_dir}/vitessce.template.json`
```

Comment:
It's possible I missed this, but there are no scripts for the creation of these templates here?  Why not make this part of the process?

Reply:
It will look slightly different for each config, and I figured it would be easier to just copy the closest Vitessce config and modify it (e.g., if you have an scRNA-seq dataset, starting from an existing scRNA-seq config, like the Satija example. If imaging-only, starting from the Spraggins example would make more sense)